frontend/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "magink",
   "version": "0.1.0",
-  "homepage": "https://swanky-dapps.github.io/magink-dapp",
+  "homepage": "https://github.com/ibrohimkhan/magink-dapp",
```

Comment:
this should be
`https://yaymalaga.github.io/magink-dapp/`

and please add this link to Repo description